src/server/game/Entities/Player/Player.cpp

```diff
@@ -1081,6 +1081,11 @@ void Player::Update(uint32 p_time)
 
     UpdateAfkReport(now);
 
+    if (IsInCombat())
```

Comment:
Is there a `m_combatTimer` equivalent for master? IIRC that's only nonzero if in PvP combat.